rct229/rules/section6/section6rule3.py

```diff
@@ -0,0 +1,93 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_building_segment_lighting_status_type_dict import (
+    LightingStatusType,
+    get_building_segment_lighting_status_type_dict,
+)
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.std_comparisons import std_equal
+
+OFFICE_OPEN_PLAN = schema_enums[
+    "LightingSpaceType2019ASHRAE901TG37"
+].OFFICE_OPEN_PLAN.name
```

Comment:
No longer need the ".name" for enumerations if you merge develop into this branch.